README.md

```diff
@@ -76,7 +76,7 @@ PartyFoul.configure do |config|
   # The OAuth token for the account that is opening the issues on GitHub
   config.oauth_token            = 'abcdefgh1234567890'
 
-  # The API endpoint for GitHub. Unless you are hosting a private
+  # The API api_endpoint for GitHub. Unless you are hosting a private
```

Comment:
I don't see the reason to change this to `api_endpoint`. That sentence isn't referencing the specific attribute but the nature of it